release/ray_release/byod/requirements_ml_byod_3.9.in

```diff
@@ -13,3 +13,11 @@ transformers
 torch
 torchtext
 torchvision
+bitsandbytes
+wandb
+pytorch-lightning
+protobuf<3.21.0
+torchmetrics
+lm_eval
+tiktoken
+sentencepiece
```

Comment:
@can-anyscale So, all release tests are running with the same docker image now? If I want to add a package to one release test, I need to add it to the common byod requirement file? Is there a doc where I can read about this?

Reply:
Yes. Tho on a PR it would still use your specified cluster_env but on the builds from master it is currently using the byod. @can-anyscale will deprecate the use of cluster envs soon for both PRs and master builds.
BYOD has a couple of benefits:
1. You don't have to build the envs when launching release tests which means faster time to failure if any
2. It will make tests more reliable due to consistency of versions. The best case scenario is if we don't pin anything in in the byod requirements files.